main.js

```diff
@@ -0,0 +1,107 @@
+// Add event listener 
+// prevent default and const for inputs
+document.getElementById('form-post').addEventListener('submit', function(e) {
+  e.preventDefault();
+
+ 
+  const textFromPost = document.getElementById('text-from-post').value;
+  const writerPosted = document.getElementById('writer-posted').value;
+  
+  postToAdd(textFromPost, writerPosted);
+
+  
+  document.getElementById('text-from-post').value = '';
+  document.getElementById('writer-posted').value = '';
```

Comment:
what is the purpose of this? 
Better practice is to extract to a function to help with readability and reusability